src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANEncoder;
+
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.HardwareMap.ShooterHardware;
+
+public class ShooterSubsystem extends SubsystemBase {
+  private SpeedControllerGroup m_shooter;
+  private SpeedControllerGroup m_feeder;
+  private CANEncoder m_canEncoder;
+  private double m_targetSpeed;
+  private double m_feederSpeed;
+  
+
+  /** Creates a new ShooterSubsystem. */
+  public ShooterSubsystem(ShooterHardware shooter) {    
```

Comment:
Please also briefly explain the distinction between the shooter and feeder in javadocs for this constructor.  Also rename `m_canEncoder` and briefly describe its purpose.